database/gormdb_assignment.go

```diff
@@ -123,6 +123,19 @@ func (db *GormDB) GetCourseAssignmentsWithSubmissions(courseID uint64, submissio
 	return filteredAssignments, nil
 }
 
+// GetCourseAssignmentsWithSubmissionsNoBuildInfo
+// returns data required for results page (score and status)
+func (db *GormDB) GetCourseAssignmentsWithSubmissionsNoBuildInfo(courseID uint64, submissionType pb.SubmissionsForCourseRequest_Type) ([]*pb.Assignment, error) {
+	var assignments []*pb.Assignment
+
```

Comment:
Prefer to avoid blank lines, unless function is long, and there are natural reasons to add some spacing.